### PR TITLE
Update deploy-intro.hmtl

### PR DESCRIPTION
**Pull request by satyampsoni**
Before:
It was not clear in the mind of the reader what actually are deployment configurations and instances.

changes:
Proposed the definition of Deployment configuration and instances guided by @sftim in my previous PR about the changes to be made. The link to PR is https://github.com/kubernetes/website/pull/40033
And the suggestion comment https://github.com/kubernetes/website/pull/40033#pullrequestreview-1343259819

After:
It gives an understanding of what deployment configuration and instances are right in the documentation.e

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 1]
```diff
@@ -27,10 +27,11 @@ <h3>Objectives</h3>
...
                 <p>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
-                To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
-                how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
-                control plane schedules the application instances included in that Deployment to run on individual Nodes in the
-                cluster.
+                To do so, you create a Kubernetes <b>Deployment</b> (a Kubernetes object that declares your intent to
+                run a set of application instances). The Deployment instructs Kubernetes
+                how to create and update instances (Copy of a Particular object) of your application. Once you've created a Deployment, the Kubernetes
+                control plane schedules those application instances associated with the that Deployment to run
```
Comment (divya-mohan0209) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:33]: Doesn't flow correctly, please rephrase.

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 2]
```diff
@@ -27,10 +27,11 @@ <h3>Objectives</h3>
                 <h3>Kubernetes Deployments</h3>
                 <p>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
-                To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
-                how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
-                control plane schedules the application instances included in that Deployment to run on individual Nodes in the
-                cluster.
+                To do so, you create a Kubernetes <b>Deployment</b> (a Kubernetes object that declares your intent to
+                run a set of application instances). The Deployment instructs Kubernetes
+                how to create and update instances (Copy of a Particular object) of your application. Once you've created a Deployment, the Kubernetes
```
Comment (divya-mohan0209) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:32]: The existing phrase works fine here. I do not see the value add.

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 3]
```diff
@@ -27,10 +27,11 @@ <h3>Objectives</h3>
                 <h3>Kubernetes Deployments</h3>
                 <p>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
-                To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
-                how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
-                control plane schedules the application instances included in that Deployment to run on individual Nodes in the
-                cluster.
+                To do so, you create a Kubernetes <b>Deployment</b> (a Kubernetes object that declares your intent to
```
Comment (divya-mohan0209) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:30]: I am not sure we can term Deployment as an object that declares the user's intent to run application instances. The existing phrasing works fine, IMO, here & I do not see the value add.

**Review by divya-mohan0209 — COMMENTED** [review]
We have sufficient documentation within our website that delves into each of these terminologies. These reside in the concept section. What exactly are we trying to add in terms of value here?

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html (hunk 3 above)
Comment (satyampsoni) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:30]: For any beginner, It was quite confusing so I simplified that.
@sftim agreed on the issue and he did suggested me these changes.
Pls have a look at it.
https://github.com/kubernetes/website/pull/40033

Comment (divya-mohan0209) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:30]: @satyampsoni : Please mention that in the description.

**Comment by natalisucks** [comment]
@satyampsoni Hi there – could you please review the rest of Divya's feedback so that we can proceed with this review? If you're unable to work on this, or if more time passes without your engagement, we'll need to close this PR. Thanks!

**Comment by satyampsoni** [comment]
Yes! I don't find the value of this PR now. Pls close it

**Pull request closed without merging**